matter/efr32/efr32mg24/BRD4186A/config/sl_device_init_hfxo_config.h

```diff
@@ -16,7 +46,7 @@
 
 // <o SL_DEVICE_INIT_HFXO_CTUNE> CTUNE <0-255>
 // <i> Default: 140
-#define SL_DEVICE_INIT_HFXO_CTUNE          140
+#define SL_DEVICE_INIT_HFXO_CTUNE          87
```

Comment:
Why did CTUNE change to non default value

Reply:
slc generation, my guess is the original value was for rev0?